### PR TITLE
Added ability to delete stories, reworked code a little bit, cleaned up website a little bit

### PR DESCRIPTION
**Pull request by cpham5**
Users are now able to delete their owned stories by pressing the delete button while editing their stories. Also, admins are able to delete the stories if needed but cannot alter them. I reworked/cleaned code a bit and cleaned up the website a bit as well.

### CODE REVIEW

File: tmasTest/templates/edit.html [hunk 1]
```diff
@@ -2,27 +2,64 @@
 
 {% block content %}
 
-<!-- Create form for editing the location, date, subject line, and story -->
-
```
Comment (cpham5) [inline tmasTest/templates/edit.html:6]: I don't know why it says I deleted this whole section. It still in there, just indented.

File: tmasTest/templates/edit.html [hunk 2]
```diff
@@ -2,27 +2,64 @@
...
+            <label>{{story.storyID}}</label><br/>
+            <label>Subject</label>
+            <textarea rows="1" cols="100" name="subject"> {{story.subject}}
+            </textarea> <br/>
+            <label>Story</label>
+            <textarea rows="5" cols="120" name="story"> {{story.story}}
+            </textarea> <br/>
+            <input type="Submit" class="center2"><br/>
+        </form>
+
```
Comment (cpham5) [inline tmasTest/templates/edit.html:29]: View story for admin

File: tmasTest/templates/edit.html [hunk 3]
```diff
@@ -2,27 +2,64 @@
...
+                {{story.story}}
+                </div>
+            </div>
+
+            <div style = "font-size: 9pt; position: float; padding-left: 90%; padding-top: 2%;">
+                {{story.location}} | {{story.community}}
+            </div>
+        
+    {%endif%}
+
```
Comment (cpham5) [inline tmasTest/templates/edit.html:56]: Delete button; might be able to make it more efficient but works for now

File: tmasTest/templates/registration/login.html [hunk 4]
```diff
@@ -6,11 +6,12 @@
 <h1 style="color:darkblue;">Login</h1>
```
Comment (cpham5) [inline tmasTest/templates/registration/login.html:7]: cleaned up page when viewed

File: tmasTest/templates/registration/signup.html [hunk 5]
```diff
@@ -3,14 +3,16 @@
 {% block title %}Sign Up{% endblock %}
```
Comment (cpham5) [inline tmasTest/templates/registration/signup.html:4]: cleaned up page when viewed

File: tmasTest/templates/tmasTest/index.html [hunk 6]
```diff
@@ -16,10 +16,7 @@
     <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
     <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
-    <style>
```
Comment (cpham5) [inline tmasTest/templates/tmasTest/index.html:19]: seemed unnecessary

File: tmasTest/templates/tmasTest/index.html [hunk 7]
```diff
@@ -29,105 +26,79 @@
     height="200px" class="center"/> <br/>
```
Comment (cpham5) [inline tmasTest/templates/tmasTest/index.html]: cleaned up code

**Comment by cpham5** [comment]
Formal tests of delete still needs to be completed, but preliminary tests seem good.

**Comment by mabdelh1** [comment]
Minor website view changes. The biggest change: added a delete button for Admins and users.

**Pull request merged**